docs/organise/before/judging.md

```diff
@@ -1,16 +1,30 @@
 # Judging
 
 ## Who should be your judges?
+The following are all groups you can select judges from, though it is recommended to sample from each pool available to you. 
 
-- Sponsors (depending on tier)
-- Some of your mentors
-- Some of your organisers
-- Try and prevent bias.
+- Sponsors (They may also be judging to give away sponsor prizes)
+- Mentors
+- Organisers
+- University Faculty
+
+When selecting judges you should aim to have enough judges to cover all of your hacks in a reasonable timeframe. Bias should be avoided if possible.  
+
+## How to structure judging?
+Allocate adequate time given your number of judges and attendees to allow the judges adequate time to consider each hack they need to judge.
+
+### Presentations
+This is the simplist approach to judging, every team presents their hack to all attendees and judges with a small, 5-10 minute, slot. Judges will see all hacks but this can become incredibly time consuming for large hackathons.
```

Comment:
I would definitely change the numbers here. Capping teams at 3 minutes is pretty common, and I'd say definitely no more than 5. If every team is presenting, you want to keep teams' presentations as short as possible.